Packs/SentryWire/Integrations/SentryWire/SentryWire.py

```diff
@@ -0,0 +1,334 @@
+            method='GET',
+            full_url=urljoin(self._base_url, endpoint),
+            params=params
+        )
+
+    def download_metadata(self, search_id: str, node_name: str):
+        endpoint = 'fnmetadata'
+        params = {
+            'searchname': search_id,
+            'nodename': node_name
+        }
+        params.update(self._session.cookies)
+        return self._http_request(
+            method='GET',
+            full_url=urljoin(self._base_url, endpoint),
+            params=params
+        )
+
+    def get_search_status(self, node_name: str, search_id: str):
+        endpoint = 'fnsearchstatus'
+        params = {
+            'searchname': search_id,
+            'nodename': node_name
+        }
+        params.update(self._session.cookies)
+        return self._http_request(
+            method='GET',
+            full_url=urljoin(self._base_url, endpoint),
+            params=params
+        )
+
+    def get_server_status(self):
+        endpoint = 'fmping'
+        return self._http_request(
+            method='GET',
+            full_url=urljoin(self._base_url, endpoint),
+            params=self._session.cookies)
+
+    def _get_auth(self) -> str:
+        endpoint = 'fmlogin'
+        json_data = {
+            'username': self._auth[0],
+            'password': self._auth[1]
+        }
+
+        response = self._http_request(
+            method='POST',
+            full_url=urljoin(self._base_url, endpoint),
+            json_data=json_data)
+
+        if isinstance(response, Dict):
+            return str(response.get('rest_token'))
+        else:
+            raise Exception(f'Authentication failed, unexpected response: {response}')
+
+
+''' HELPER FUNCTIONS '''
+
+
+def bytes_to_readable(response):
+    # Search was cancelled
+    if response["SearchResult"] == "Cancelled":
+        readable_output = "Search was cancelled"
+        response["SearchStatus"] = "Cancelled"
+        response["SearchResult"] = "NoPcapData"
+    # Search was completed with no pcap data
+    elif response["SearchResult"] == "NoPcapData":
+        readable_output = "Search completed: No results found"
+        response["SearchStatus"] = "Completed"
+    else:
+        # Default
+        results = response["SearchResult"]
+        sizestr = response["SearchResult"].split(" ")[-1].split("=")[-1]
+        size, unit = re.match(r"(\d+)(\D+)", sizestr).groups()
+        if unit == "KB":
+            response["PcapSize"] = int(size) * 1024
+        elif unit == "MB":
+            response["PcapSize"] = int(size) * (1024 ** 2)
+        elif unit == "GB":
+            response["PcapSize"] = int(size) * (1024 ** 3)
+        else:
+            response["PcapSize"] = ">=1TB"
+        readable_output = f"Search completed: {results}"
+        response["SearchStatus"] = "Completed"
+    return response, readable_output
+
+
+''' COMMANDS '''
+
+
+def create_search_command(client: Client, args: Dict[str, Any]):
+    generate_links = False

```

Comment:
This will always be **False**. You can do:
```suggestion
    generate_links = argToBoolean(args.get('generate_links'))

```